components/forms/RadioGroup/react.js

```diff
@@ -0,0 +1,42 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import { React as Radio } from 'components/forms/RadioButton'
+import s from './style.module.scss'
+
+const RadioGroup = ({
+  checkedChoice, choices, direction, onChange,
+}) => {
+  const radios = choices.map(({ label, value }) => (
+    <li key={value} className={direction === 'column' ? '' : s['radio-group__row']}>
+      <Radio
+        checked={checkedChoice === value}
+        onClick={onChange}
+        onChange={onChange}
+        label={label}
+        value={value}
+      />
+    </li>
```

Comment:
I was suggesting placing the label text here, so `<li><label><Radio /><span className={someClass}>{label}</span></label></li>`, classed for a margin.

Reply:
Fixed